Kevin Lin Headstorm Interview code/BackEnd/app.js

```diff
@@ -0,0 +1,47 @@
+// backend rest api implemented using 
+// Node JS (express framework), nodemon, 
+// (with Postman http, iTerm2)
+const express = require('express');
+const app = express();
+
+// app.use((req, res, next) => {
+//     res.status(200).json({
+//         message: 'works!'
+//     });
+// });
+
+// app.get('/', (req, res) => {
+//     res.send('hello world!');
+// });
+
+// app.get('/api/courses', (req, res) => {
+//     res.send([1,2,3,4]);
+// });
+
+app.post('/data', (req, res) => {
+    if (!req.body.name || req.body.name.length < 3) {
+        // 400 bad request 
+        res.status(400).send('JSON needs exactly 500 random numbers (double, int)')
+    }
+})
+
+app.get('/data', (req, res) => {
+    
```

Comment:
I don't see any data/response returned in this block. How are the numbers being returned to the client?